config/routes.rb

```diff
@@ -154,6 +118,40 @@
     resources :sites
   end
 
+  get '/attachment/:id/download(/:name)' => 'attachments#download',
+      as: :download_attachment
+  get '/attachment/:id/show' => 'attachments#show',
+      as: :show_attachment
+
+  # Picture urls
+  get "/pictures/:id/show(/:size)(/:crop)(/:crop_from/:crop_size)(/:quality)/:name.:format" => 'pictures#show',
```

Comment:
Metrics/LineLength: Line is too long. [111/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)